### PR TITLE
Add experimental summarizers

### PR DESCRIPTION
**Pull request by pwoznic**
Added the two new beta summarizers

### CODE REVIEW

File: www/docs/learn/grounded-generation/summarizers.md [hunk 1]
```diff
@@ -8,17 +8,62 @@ import {Config} from '@site/docs/definitions.md';
...
+summarizer config at request time, <Config v="names.product"/> uses the best
+available summarizer for your account. 
+
+## Currently Available Summarizers
+
+Currently, we have two official summarizers available to our users. Growth 
+users have `vectara-summary-ext-v1.2.0` (GPT 3.5), while Scale users have 
+access to `vectara-summary-ext-v1.3.0` (GPT 4.0). 
+
+## Experimental Summarizers
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:28]: Maybe say "Experimental (beta) Summarizers"?

File: www/docs/learn/grounded-generation/summarizers.md [hunk 2]
```diff
@@ -8,17 +8,62 @@ import {Config} from '@site/docs/definitions.md';
...
+
+We also have two new experimental beta summarizers available only for
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:30]: Oh I see you used "beta" here. I personally prefer that in the title and then here to just use "two new beta summarizers..."
the logic is to connect beta to experimental.

File: www/docs/learn/grounded-generation/summarizers.md [hunk 3]
```diff
@@ -8,17 +8,62 @@ import {Config} from '@site/docs/definitions.md';
...
+
+* `vectara-experimental-summary-ext-2023-10-23-small` (GPT 3.5) 
+* `vectara-experimental-summary-ext-2023-10-23-med` (GPT 4.0)
+
+These beta versions are a preview of our next improved summarizers. Since 
+they are experimental, we do not support them officially.
+
+### Experimental Summarizer Example
+
+The following example query selects the experimental GPT 4.0 summarizer:
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:41]: GPT 4.0 summarizer (only available to Scale customers):

File: www/docs/learn/grounded-generation/summarizers.md [hunk 4]
```diff
@@ -23,11 +22,11 @@ available summarizer for your account.
...
 Currently, we have two official summarizers available to our users. Growth 
 users have `vectara-summary-ext-v1.2.0` (GPT 3.5), while Scale users have 
-access to `vectara-summary-ext-v1.3.0` (GPT 4.0). 
+access to `vectara-summary-ext-v1.3.0` (GPT 4.0).
 
-## Experimental Summarizers
+## Beta Summarizers
 
-We also have two new experimental beta summarizers available only for
+We also have two new beta summarizers available only for
```
Comment (ofermend) [inline www/docs/learn/grounded-generation/summarizers.md:29]: Looks better. I think the vectara-experimental-summary-ext-2023-10-23-small is available to anyone (even growth users) whereas the "med" (GPT4) is available for anyone.

Comment (ofermend) [reply www/docs/learn/grounded-generation/summarizers.md]: yes sorry I meant to say "Med" is available for Scale only.

File: www/docs/learn/grounded-generation/summarizers.md [hunk 5]
```diff
@@ -8,30 +8,71 @@ import {Config} from '@site/docs/definitions.md';
...
-available to Growth users is `vectara-summary-ext-v1.2.0`.
+Summarizers have both names and IDs, and are versioned. Providing the
+summarizer as part of the config is optional. If you do not provide a
+summarizer config at request time, <Config v="names.product"/> uses the best
+available summarizer for your account. 
 
+## Currently Available Summarizers
+
+Currently, we have two official summarizers available to our users. Growth 
+users have `vectara-summary-ext-v1.2.0` (GPT 3.5), while Scale users have
```
Comment (eskibars) [inline www/docs/learn/grounded-generation/summarizers.md:24]: I think we should expand on this a bit before diving straight into what's available.  e.g. first stating what the versioning scheme is ("Prompt names have a prefix and a version.  They encapsulate both a prompt text, as well as potentially specific configuration options for the generative system.  `vectara-summary-ext` is the prefix for generative summarization of the results.  Today, the versions available are `1.2.0` which uses chatgpt-3.5-turbo and `1.3.0` which uses gpt-4.0 and is only available to our paying Scale customers.  1.2.0 is typically faster while 1.3.0 is typically slower but produces a more accurate summary.")

File: www/docs/learn/grounded-generation/summarizers.md [hunk 6]
```diff
@@ -8,30 +8,71 @@ import {Config} from '@site/docs/definitions.md';
...
+
+## Beta Summarizers
+
+We also have two new beta summarizers available for our users to try:
+
+* **Growth and Scale:** `vectara-experimental-summary-ext-2023-10-23-small` (GPT 3.5)
+* **Scale only:** `vectara-experimental-summary-ext-2023-10-23-med` (GPT 4.0)
+
+These beta versions are a preview of our next improved summarizers. Since 
+they are experimental, we do not support them officially.
```
Comment (eskibars) [inline www/docs/learn/grounded-generation/summarizers.md:35]: We should add that while we don't support them officially, we are currently considering promoting them to GA, pending feedback from our users

File: www/docs/learn/grounded-generation/summarizers.md [hunk 7]
```diff
@@ -8,30 +8,71 @@ import {Config} from '@site/docs/definitions.md';
...
 
-The default limit of `maxSummarizedResults` is `10` search results for Growth 
-plans and this limit can be extended for Scale plan users. Setting the values 
-closer to the limit generates a more comprehensive summary, but using a lower 
-value can balance the results with quality and response time.
+The default limit of maxSummarizedResults is 10 search results for Growth plans and this limit can be extended for Scale plan users. Setting the values closer to the limit generates a more comprehensive summary, but using a lower value can balance the results with quality and response time.
 
-## maxSummarizedResults Example
+maxSummarizedResults Example
+This summarizer example attempts to balance creating a good quality summary with a a reasonably fast response by setting maxSummarizedResults to 5. To use vectara-summary-ext-v1.2.0, send it as the summarizerPromptName as follows:
```
Comment (eskibars) [inline www/docs/learn/grounded-generation/summarizers.md:71]: You have a double "a a" here

File: www/docs/learn/grounded-generation/summarizers.md [hunk 8]
```diff
@@ -8,30 +8,71 @@ import {Config} from '@site/docs/definitions.md';
...
 
-This summarizer example attempts to balance creating a good quality summary with a
-a reasonably fast response by setting `maxSummarizedResults` to `5`. To use `vectara-summary-ext-v1.2.0`, send 
-it as the `summarizerPromptName` as follows:
+This summarizer attempts to balance creating a good quality summary with a
+a reasonably fast response.  To use `vectara-summary-ext-v1.2.0`, send it as
```
Comment (eskibars) [inline www/docs/learn/grounded-generation/summarizers.md:74]: Double "a" here as well (with the previous line)

**Review by eskibars — CHANGES_REQUESTED** [review]

**Comment by pwoznic** [comment]
Updated topic after reviewing comments

**Pull request merged**